include/podio/CollectionBufferFactory.h

```diff
@@ -1,14 +1,15 @@
 #ifndef PODIO_COLLECTIONBUFFERFACTORY_H
 #define PODIO_COLLECTIONBUFFERFACTORY_H
 
-#include "podio/CollectionBuffers.h"
-#include "podio/SchemaEvolution.h"
-
 #include <functional>
 #include <optional>
+#include <string>
 #include <unordered_map>
 #include <vector>
 
+#include "podio/CollectionBuffers.h"
+#include "podio/SchemaEvolution.h"
+
```

Comment:
Can we keep "our" headers above the STL ones? Or did something change here for the recommendations? (Mainly because this makes it more likely to miss some STL / system headers in our included files)

Reply:
Fixed by changing .clang-format, at least the headers I checked had the system ones below.